bin/runtime/src/lib.rs

```diff
@@ -370,6 +372,63 @@ impl pallet_staking::EraPayout<Balance> for UniformEraPayout {
     }
 }
 
+type SubstrateStakingWeights = pallet_staking::weights::SubstrateWeight<Runtime>;
+
+pub struct PayoutStakersDecreasedWeightInfo(pallet_staking::weights::SubstrateWeight<Runtime>);
+impl pallet_staking::WeightInfo for PayoutStakersDecreasedWeightInfo {
+    wrap_method!(bond(), SubstrateStakingWeights, Weight);
+    wrap_method!(bond_extra(), SubstrateStakingWeights, Weight);
+    wrap_method!(unbond(), SubstrateStakingWeights, Weight);
+    wrap_method!(
+        withdraw_unbonded_update(s: u32),
+        SubstrateStakingWeights,
+        Weight
+    );
+    wrap_method!(
+        withdraw_unbonded_kill(s: u32),
+        SubstrateStakingWeights,
+        Weight
+    );
+    wrap_method!(validate(), SubstrateStakingWeights, Weight);
```

Comment:
how hard would it be to make it:
```rust
wrap_methods!(
    (bond(), SubstrateStakingWeights, Weigh),
    (bond_extra(), SubstrateStakingWeights, Weigh),
    ...
 )
```

Reply:
done, good point. Also added docs to that macro.